students/Lincoln_Zhang/Session4/test_trigrams.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+import trigrams,os
+
+global_result = trigrams.read_file("sherlock_small.txt")
+
+def test_read_file():
+	assert global_result[0] == "One"
+	assert global_result[1] == "night"
+
+def test_build_trigram():
+	local_result = trigrams.build_trigram(global_result)
+	assert type(local_result) == dict
```

Comment:
you generally don't want to test that you get the right type -- do you care? what you want is an object with the right interface -- that is, use the result, and see if it works. (like you have in the following lines)